examples/remarketing/add_remarketing_action.py

```diff
@@ -0,0 +1,143 @@
+#!/usr/bin/env python
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""This example demonstrates usage of remarketing actions.
+
+A new remarketing action will be created for the specified customer, and its
+associated tag snippets will be retrieved.
+"""
+
+
+import argparse
+import sys
+import uuid
```

Comment:
suggest you make this `from uuid import uuid4` and then reference just `uuid4()`